types/http-errors/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for http-errors 1.8
+// Type definitions for http-errors 2.0
```

Comment:
`UnorderedCollection` was renamed to `TooEarly` in 2.0

Reply:
Thanks, I've renamed `UnorderedCollection` to `TooEarly`.